pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "mpt-extension-sdk"
-version = "0.1.0"
+version = "4.1.1"
```

Comment:
It shouldn't be 0.1.1, should it?

Reply:
This value has never been updated, actually. I decided to update it because it was creating confusion showing up in certain places when testing locally